packrat/src/main/java/com/github/reflectoring/infiniboard/packrat/dashboard/DashboardConfig.java

```diff
@@ -0,0 +1,30 @@
+package com.github.reflectoring.infiniboard.packrat.dashboard;
+
+import org.springframework.data.annotation.Id;
+
+/**
```

Comment:
java doc can be removed. does not explain anything which is not already contained in class name.